cmd/img_mgmt/include/img_mgmt/img_mgmt_config.h

```diff
@@ -32,13 +32,41 @@
 
 #elif defined __ZEPHYR__
 
+#include <devicetree.h>
+
 #define IMG_MGMT_UL_CHUNK_SIZE  CONFIG_IMG_MGMT_UL_CHUNK_SIZE
 #define IMG_MGMT_VERBOSE_ERR    CONFIG_IMG_MGMT_VERBOSE_ERR
 #define IMG_MGMT_LAZY_ERASE     CONFIG_IMG_ERASE_PROGRESSIVELY
 #define IMG_MGMT_DUMMY_HDR      CONFIG_IMG_MGMT_DUMMY_HDR
-#define IMG_MGMT_BOOT_CURR_SLOT 0
 
+#define DT_CODE_PARTITION_NODE DT_CHOSEN(zephyr_code_partition)
```

Comment:
I wonder whether this can be included as a zephyr-rtos header?

Reply:
You mean that it should be in separate header taken directly from Zrephyr code base?
I was also thinking about that and that is one of reasons behind https://github.com/zephyrproject-rtos/zephyr/issues/32257